eox_tenant/middleware.py

```diff
@@ -126,3 +92,16 @@ def process_request(self, request):
                     {'domain': domain, }
                 )
             )
+
+
+class MonkeyPatchMiddleware(object):
```

Comment:
I don't know so much about django monkey patching best practices but what about monkeypatching the SiteConfiguration model after Django load? does it pose some technical problem?

Reply:
@jfavellar90 I don't know if monkey patch has best practices, since the most of people recommends don't use that,anyway  another good place to do this is in the apps.py  in the method ready, the problem, in that point, is that the settings has not been loaded yet, as we use the setting module to load all the backends, we could not get the backend in the standard way, another reason to make this in the middleware was this article https://opensourcehacker.com/2010/08/12/applying-monkey-patches-in-django-middleware-layer/

Reply:
We moved this to the ready method of the apps config